pkg/renderer/sgml/section.go

```diff
@@ -0,0 +1,89 @@
+package sgml
```

Comment:
shouldn't this be part of the `html5` subpackage?

Reply:
I don't think so.  Why do you?

Reply:
ah never mind, I wasn't awake 🤦‍♂️